### PR TITLE
Use on() in place of live() in impala/users

### LINKED ISSUE #1253: Replace live() in impala/users
Relates to #1197

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #1253

### CODE REVIEW

File: static/js/impala/users.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ $(function() {
 
         var required = 'input[aria-required=true], select[aria-required=true]';
 
-        $form.live('change input', required, validate_required_fields);
+        $form.on('change input', required, validate_required_fields);
```
Comment (muffinresearch) [inline static/js/impala/users.js]: This is pretty self-contained code so I think this doesn't need to be delegated.

Comment (muffinresearch) [inline static/js/impala/users.js]: Hacking the views to test it the code enables/disables the submit as expected.

**Comment by tofumatt** [comment]
r+

**Comment by magopian** [comment]
r+

**Pull request merged**